regolith/helpers/l_projectahelper.py

```diff
@@ -118,13 +123,17 @@ def sout(self):
         end_projecta = []
         grouped_projecta = {}
         if rc.lead and rc.person:
-            raise RuntimeError(f"please specify either lead or person, not both")
+            raise RuntimeError(
+                f"please specify either lead or person, not both")
         for projectum in collection:
-            if rc.all and projectum.get('status') != "finished":
-                projecta.append(projectum)
-                continue
             if isinstance(projectum.get('group_members'), str):
                 projectum['group_members'] = [projectum.get('group_members')]
+            if rc.all and (projectum.get('status') == True):
+                projecta.append(projectum)
+                continue
+            if rc.current and (projectum.get('status') == "proposed" or "started"):
+                projecta.append(projectum)
+                continue
```

Comment:
doing it this way will mean that none of the other filters will work.  Is that the desired behavior?  I don't remember the discussion.